### ISSUE TITLE
solver files for CUB-200-2011

### ISSUE THREAD

**Issue by vadimkantorov** [rksltnl/Deep-Metric-Learning-CVPR16#11]
Hi,

I noticed that the pretrained models for CUB have baselr_1E-6 in their names. Have you used learning rate 1e-6 for this dataset? If possible, could you post the parameters (or whole solver files) used for training on CUB?

Thanks,
Vadim

**Comment by rksltnl** [comment]
Here's the solver I used for the birds dataset.
net: "/INSTALL_PATH/Deep-Lifting-for-Metric-Learning-CVPR/code/birds/model/train_val_googlenet_finetune_liftedstructsim_softmax_pair_m128_multilabel_embed64.prototxt"
test_initialization: false
\# lr for fine-tuning should be lower than when starting from scratch
base_lr: 0.000001
lr_policy: "step"
gamma: 0.96
\# stepsize should also be lower, as we're closer to being done
stepsize: 32000
display: 5
max_iter: 40000
momentum: 0.9
weight_decay: 0.0002
snapshot: 10000
snapshot_prefix: "snapshot_birds_googlenet_finetune_liftedstructsim_softmax_pair_m128_multilabel_embed64_baselr1E6"
solver_mode: GPU

Here's the solver I used for the cars dataset.
net: "/INSTALL_PATH/Deep-Lifting-for-Metric-Learning-CVPR/code/cars196/model/train_val_googlenet_finetune_liftedstructsim_softmax_pair_m128_multilabel_embed64.prototxt"
test_initialization: false
\# lr for fine-tuning should be lower than when starting from scratch
base_lr: 0.0001
lr_policy: "step"
gamma: 0.96
\# stepsize should also be lower, as we're closer to being done
stepsize: 32000
display: 5
max_iter: 40000
momentum: 0.9
weight_decay: 0.0002
snapshot: 10000
snapshot_prefix: "snapshot_cars_googlenet_finetune_liftedstructsim_softmax_pair_m128_multilabel_embed128"
solver_mode: GPU
